plugins/producer_api_plugin/producer.swagger.yaml

```diff
@@ -22,477 +23,607 @@ servers:
       port:
         default: "8080"
-components:
-  securitySchemes: {}
-  schemas: {}
 security:
   - {}
 paths:
   /producer/pause:
     post:
       summary: pause
-      description: Pause producer node
+      description: Pause producer node. Takes no arguments and returns no values.
       operationId: pause
-      parameters: []
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              properties: {}
       responses:
-        "200":
+        "201":
           description: OK
           content:
             application/json:
               schema:
-                type: boolean
-                description: "returns status"
+                $ref: '#/components/schemas/OK'
+        "400":
+          description: client error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
   /producer/resume:
     post:
       summary: resume
-      description: Resume producer node
+      description: Resume producer node. Takes no arguments and returns no values.
       operationId: resume
-      parameters: []
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              description: Resumes activity for producer
       responses:
-        "200":
+        "201":
           description: OK
           content:
             application/json:
               schema:
-                description: Returns Nothing
+                $ref: '#/components/schemas/OK'
+        "400":
+          description: client error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
   /producer/paused:
     post:
       summary: paused
-      description: Retreives paused status for producer node
+      description: Retrieves paused status for producer node. Takes no arguments and returns no values.
       operationId: paused
-      parameters: []
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              properties: {}
       responses:
-        "200":
+        "201":
           description: OK
           content:
             application/json:
               schema:
-                type: boolean
-                description: True if producer is paused, false otherwise
+                $ref: '#/components/schemas/OK_PAUSED'
+        "400":
+          description: client error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
   /producer/get_runtime_options:
     post:
       summary: get_runtime_options
-      description: Retreives run time options for producer node
+      description: Retrieves runtime options for producer node. `incoming_defer_ratio` is a floating point number. The other return values are integers.
```

Comment:
Probably not a good idea to mention data types on description. It creates another source of truth dependency. For instance, this statement might be incorrect in the future: The other return values are integers.